From: Adrian Bunk <bunk@debian.org>
Bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1786619
Bug-Debian: https://bugs.debian.org/1017961

SIMD (SSE/NEON) is not in the baseline of the i386/armel/armhf ports.

Index: mozjs102-102.2.0/js/moz.configure
===================================================================
--- mozjs102-102.2.0.orig/js/moz.configure
+++ mozjs102-102.2.0/js/moz.configure
@@ -573,7 +573,7 @@ option(
 def rust_simd(value, target):
     # As of 2019-09-17, the simd-accel feature of encoding_rs has not
     # been properly set up outside aarch64, armv7, x86 and x86_64.
-    if target.cpu in ("aarch64", "arm", "x86", "x86_64") and value:
+    if target.cpu in ("aarch64", "x86_64") and value:
         return True
 
 
